src/main/java/frc/states/MechStates/ReadyForIntake.java

```diff
@@ -0,0 +1,21 @@
+package frc.states.mechstates;
+
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.LauncherSubsystem;
+import frc.states.MechState;
+
+public class ReadyForIntake extends MechState {
+
+    private final IntakeSubsystem mIntakeSubsystem;
+
+
+    public ReadyForIntake (IntakeSubsystem intakeSubsystem, LauncherSubsystem launcherSubsystem) {
+        super(launcherSubsystem);
+        mIntakeSubsystem = intakeSubsystem;
+    }
+
+    @Override
+    public void intake() {
+        mIntakeSubsystem.getAutomousIntakeCommand();
+    }
```

Comment:
This applies to the other autonomous intake command in the ReadyForHandoff class too. You are simply retrieving the intake command rather than scheduling it
```suggestion
        mIntakeSubsystem.getAutomousIntakeCommand().schedule();
    }
```